week-4/E-arrays-of-objects/exercise-1.js

```diff
@@ -25,11 +25,24 @@ WRITE YOUR CODE BELOW
 */
 
 
-var persons = // Complete here
+var persons = [person1, person2, person3];
+/// Complete here
 
-var personNames = // Complete here
+var personNames = persons.map(getNames);
 
-var personsYoungerThan28YearsOld = // Complete here
+function getNames (person){
```

Comment:
This function returns *one* name, so it should be named `getName`, not `getNames`